### PR TITLE
Revise test utils error messages

### PR DESCRIPTION
**Pull request by feuGeneA**
Per https://www.notion.so/nomiclabs/Hardhat-Test-Utils-Error-Messages-Copy-Review-a43c4461c9d64861bd5759149a1d6298

### CODE REVIEW

File: packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts [hunk 1]
```diff
@@ -32,11 +33,13 @@ export function supportChangeEtherBalances(Assertion: Chai.AssertionStatic) {
             actualChanges.forEach((change, i) => {
               if (!change.eq(BigNumber.from(balanceChanges[i]))) {
                 lines.push(
-                  `Expected ${
+                  `Expected ether balance of ${
```
Comment (fvictorio) [inline packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts:36]: Shouldn't this be _the_ ether balance, like in `changeEtherBalance`?

File: packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts [hunk 2]
```diff
@@ -47,27 +50,29 @@ export function supportChangeEtherBalances(Assertion: Chai.AssertionStatic) {
             actualChanges.forEach((change, i) => {
               if (change.eq(BigNumber.from(balanceChanges[i]))) {
                 lines.push(
-                  `Expected ${
+                  `Expected ether balance of ${
```
Comment (fvictorio) [inline packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts:53]: "the"?

File: packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts (hunk 1 above)
Comment (feuGeneA) [reply packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts]: Great catch, thank you. Addressed in https://github.com/NomicFoundation/hardhat/pull/2893/commits/b7051adf1189374e1d21605e6a7c42d46f78c887

File: packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts (hunk 2 above)
Comment (feuGeneA) [reply packages/hardhat-chai-matchers/src/internal/changeEtherBalances.ts]: Addressed in https://github.com/NomicFoundation/hardhat/pull/2893/commits/b7051adf1189374e1d21605e6a7c42d46f78c887

**Review by fvictorio — APPROVED** [review]

**Pull request merged**